src/Components/App/App.js

```diff
@@ -9,27 +9,40 @@ class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies,
+      movies: [],
       movieInfo: null,
+      video: [],
```

Comment:
It looks like in your code, you are using a `find` to set this property to a single video, rather than an array of videos. Because of that, I wouldn't use an empty array as the default value. I would use `null`. I tend to only use empty arrays for a variable that will later be an array. In this case, `video` is never an array, so it doesn't make sense to use one here.

Reply:
Oh that's a good thought. We felt it unnecessary to have several videos and just wanted a Trailer type video to display, which was the reasoning behind the find. But setting the value to null definitely makes more sense than an empty array. Thank you Kayla!